app/src/main/java/org/p2p/wallet/send/ui/main/SendPresenter.kt

```diff
@@ -152,6 +152,14 @@ class SendPresenter(
     override fun setTargetResult(result: SearchResult?) {
```

Comment:
you can remove ? from this method, and just avoid calling this method if result is null, cause inside this function you anyway checking for nullable

Reply:
It can't be changed cause in SearchFragment we call this functions with null param
```
            clearImageView.setOnClickListener {
                presenter.setTargetResult(null)
            }
```